draft-irtf-cfrg-opaque.md

```diff
@@ -585,18 +590,21 @@ Steps:
 ### CreateRegistrationResponse {#create-reg-response}
 
 ~~~
-CreateRegistrationResponse(request, server_public_key)
+CreateRegistrationResponse(request, server_public_key, cred_identifier, oprf_seed)
 
 Input:
 - request, a RegistrationRequest structure
 - server_public_key, the server's public key
+- cred_identifier, an identifier that uniquely represents the credential being
+  registered
+- oprf_seed, the server-side seed of Nh bytes used to generate an oprf_key
 
 Output:
 - response, a RegistrationResponse structure
 - oprf_key, the per-client OPRF key known only to the server
 
 Steps:
-1. (oprf_key, _) = GenerateKeyPair()
+1. oprf_key = HKDF-Expand(oprf_seed, concat("oprf key", cred_identifier), Nok)
```

Comment:
I don't think this is a workable replacement. This basically type casts bytes to an OPRF private key. I think we should revert this, and instead use DeriveKeyPair(HKDF(...)).

Reply:
Addressed in the latest push